src/hooks/hipchat.js

```diff
@@ -0,0 +1,50 @@
+const Hook = require("../hook");
```

Comment:
Stable. Will only be editing line 38 in service of redoing mxcontrol related files.